tests/test_integration.py

```diff
@@ -28,25 +29,34 @@ def make_factory():
     automap = automap_base(bind=db.engine, metadata=metadata)
     automap.prepare(db.engine, reflect=True)
 
-    class SchedAFactory(SQLAlchemyModelFactory):
+    class NmlSchedAFactory(SQLAlchemyModelFactory):
         class Meta:
             sqlalchemy_session = db.session
             model = automap.classes.nml_sched_a
         form_tp_cd = '11'
         contb_receipt_dt = datetime.datetime(2016, 1, 1)
         sub_id = factory.Sequence(lambda n: n)
         rpt_yr = 2016
+        amndt_ind = 'A'
 
-    class SchedBFactory(SQLAlchemyModelFactory):
+    class NmlSchedBFactory(SQLAlchemyModelFactory):
         class Meta:
             sqlalchemy_session = db.session
             model = automap.classes.nml_sched_b
         form_tp_cd = '11'
         disb_dt = datetime.datetime(2016, 1, 1)
         sub_id = factory.Sequence(lambda n: n)
         rpt_yr = 2016
+        amndt_ind = 'A'
```

Comment:
The relevant query fragments for the views are:
```
 WHERE sa.sub_id = fi.sub_id AND sa.amndt_ind::text <> 'D'::text AND sa.delete_ind IS NULL;
```
and
```
 WHERE sb.sub_id = fi.sub_id AND sb.amndt_ind::text <> 'D'::text AND sb.delete_ind IS NULL;
```

The value cannot be `D`. I chose `A` because I found rows with that value in the `dev` DB.